src/style.css

```diff
@@ -1,3 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
It is also considered good practice to add another font as a fallback font for those operating systems in which the main font is not supported